### DISCUSSION TITLE
Apexchart card take 4 columns in section Dashboard

### DISCUSSION THREAD

**Discussion by yozart** [RomRider/apexcharts-card#787] [Q&A]
Hello,

In a dashboard Section, I have added ApexChart-Card. How cn Imake it take the full width ofthe 4 columns in this case ?

![image](https://github.com/user-attachments/assets/f682b36a-d9ed-4b35-83bf-f51551da7230)

**Comment by Rayn93** [comment]
@yozart, you could change it in panel config. Use view type as:  **Panel (1 card)** 

![Screenshot 2024-10-10 at 23 02 46](https://github.com/user-attachments/assets/e760cd61-a0ac-46ec-a8e6-57fb81458fe2)

**Comment by yozart** [comment]
Well, this solution is working if you want to use a single chart on the page.

If I understand well what you're not saying: ApexChart is not compatible with "Sections" dashboard (yet; at least)

Thanks for the solution, I'll try to work with this for now and hope that Sections dashboard will be supported soon :D

**Comment by hiatsu0** [comment]
I attempted the same when trying the new Sections type for one dashboard: At the top I wanted a full width graph and underneath it some smaller sections for details. But indeed Apex card does not seem to want to scale to full width. Hoping this would be resolved at some point.
